src/components/assessment-top-wrecker-card/ASAssessmentTopWreckerCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Image, Text, View } from 'react-native'
+import { Bar } from 'react-native-progress'
+
+import { PROGRESS_CARD_IMAGE_SOURCE, ResizeMode } from '../../constants'
+import { IProgressProps } from '../../types'
+import { COLORS, SPACING } from '../../theme'
+
+import { styles } from './asAssessmentTopWreckerCard-styles'
+
+const ASAssessmentTopWreckerCard = (props: IProgressProps) => {
```

Comment:
component name is 'ASAssessmentTopWreckerCard', so the interface name should be 'IASAssessmentTopWreckerCardProps'